src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -6,18 +6,19 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.1-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.1-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.3-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.3-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="System.IO.Pipelines.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.1'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.3'">
     <Reference Include="System.Runtime" />
     <Reference Include="System.Threading.Tasks" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\System.Memory\ref\System.Memory.csproj" />
     <ProjectReference Include="..\..\System.Buffers\ref\System.Buffers.csproj" />
+    <ProjectReference Include="..\..\System.Threading.Tasks.Extensions\ref\System.Threading.Tasks.Extensions.csproj" />
```

Comment:
Are ProjectReferences needed here, or can these instead be References?

Reply:
I have no idea how the build system works. I was trying to copy what channels was doing but I know there are some issue ms we currently have

Reply:
I think they do need to be project references.
See https://github.com/dotnet/corefx/pull/27644#issuecomment-369841835

> Yes, now I remember. If you use project reference, the build order (based on dependencies) is maintained. If you use Reference, then you have to ensure (somehow) that System.Memory gets built before System.IO.Pipelines, or you will see the above errors.

Reply:
Hmm. Actually, I don't see any changes to the ref that would require this addition.

Reply:
> I have no idea how the build system works. I was trying to copy what channels was doing but I know there are some issue ms we currently have

@joperezr, @weshaggard can you please help review the csproj/ref changes? Thanks!

Reply:
> Are ProjectReferences needed here, or can these instead be References?

Usually on ref projects we use ProjectReferences, since we can't really ensure that System.Threading.Tasks.Extensions will be built before this project. On src projects on the other hand, it's ok to just use Reference, since the targeting packs would be built already so no need to rebuild them.

> Hmm. Actually, I don't see any changes to the ref that would require this addition.

If in fact this new ref is needed, then this is the right change, if we don't really use the api then we should just remove this line.